FirebaseAdmin/FirebaseAdmin.Tests/Auth/ImportUserRecordArgsTest.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace FirebaseAdmin.Auth.Tests
+{
+    public class ImportUserRecordArgsTest
+    {
+        [Fact]
+        public void TestImportUserRecordArgsSerializationBasic()
+        {
+            var userProviders = new List<UserProvider>
+            {
+                new UserProvider()
+                {
+                    Uid = "google.uid",
+                    ProviderId = "google.com",
+                },
+            };
+
+            var customClaims = new Dictionary<string, object>()
+            {
+                { "admin", true },
+            };
+
+            var userMetadata = new UserMetadata(1, 2, null);
+
+            var importUserRecordArgs = new ImportUserRecordArgs()
+            {
+                Uid = "123",
+                Email = "example@gmail.com",
+                EmailVerified = true,
+                DisplayName = "Example",
+                PhotoUrl = "http://example.com/photo",
+                PhoneNumber = "+11234567890",
+                Disabled = false,
+                UserMetadata = userMetadata,
+                PasswordHash = Encoding.ASCII.GetBytes("secret"),
+                PasswordSalt = Encoding.ASCII.GetBytes("salt"),
+                CustomClaims = customClaims,
+                UserProviders = userProviders,
+            };
+
+            var expected = new Dictionary<string, object>()
+            {
+                { "createdAt", userMetadata.CreationTimestamp },
+                { "customAttributes", JsonConvert.SerializeObject(customClaims) },
+                { "disabled", false },
+                { "displayName", "Example" },
+                { "email", "example@gmail.com" },
+                { "emailVerified", true },
+                { "lastLoginAt", userMetadata.LastSignInTimestamp },
+                { "passwordHash", "c2VjcmV0" },
+                { "salt", "c2FsdA" },
+                { "phoneNumber", "+11234567890" },
+                { "photoUrl", "http://example.com/photo" },
+                { "providerUserInfo", userProviders },
+                { "localId", "123" },
+            };
+
+            Assert.Equal(
+              JsonConvert.SerializeObject(expected),
+              JsonConvert.SerializeObject(importUserRecordArgs.ToRequest()));
+        }
+
```

Comment:
Lets also add a test case where `ImportUserRecordArgs` is initialized with the bare minimal set of properties (I think just Uid).